lib/ansible/modules/cloud/amazon/ec2_vpc_route_table.py

```diff
@@ -392,6 +402,15 @@ def ensure_routes(vpc_conn, route_table, route_specs, propagating_vgw_ids,
                 if e.error_code == 'DryRunOperation':
                     pass
 
+        for route_spec in route_specs_to_recreate:
+            try:
```

Comment:
You can delete this line (and the `except....`) as EC2ResponseError triggered by this function will be caught in main()